smartsim/entity/application.py

```diff
@@ -130,6 +130,14 @@ def files(self, value: t.Optional[EntityFiles]) -> None:
         """
         self._files = copy.deepcopy(value)
 
+    def add_exe_args(self, args: t.Union[str, t.List[str], None]) -> None:
```

Comment:
Super small suggestion, but since you moved this method would it be best placed underneath ``exe_args`` setter?

Reply:
sure!